### DISCUSSION TITLE
Button: "Change to Post Race Show" after the race is over.

### DISCUSSION THREAD

**Discussion by tonogdlp** [f1multiviewer/issue-tracker#33] [Feature requests]
I hate that, in f1tv.com I'm always forgetting that I have to go back to the main menu, then open the new 'event' which is 'Post-Race Show'. 

It will be a great add to this app if you could change the main feed to the Post Race show with an integrated button. 

I imagine that, if you have Live Timing opened, you can hunt for Race Control showing the 'Checkered Flag' and start checking every minute or so if there is a new 'Live Event'. When you find a new Live Event, you can show a button to go to it

**Comment by f1multiviewer** [comment]
Do you think that desktop notifications for new live events would solve this?

**Reply by that1polish** [reply]
in my opinion this might work better since the post race show doesnt have live timing. Possible flow if this would look like this: Race ends, notification to move to post race show is displayed as an OS notification or a window within the main race feeds window, user clicks on notification, race layout closes and post-race show is played

**Comment by AtlAntA118** [comment]
If you watch the F1live stream then it switches automatically to the post race show. International doesnt afaik.
